apps/password-reset/src/_styles.scss

```diff
@@ -78,7 +78,14 @@ $password-reset-theme: mat.define-light-theme(
   $user-primary: mat.define-palette($user-dynamic-colors, 500);
   $user-accent: mat.define-palette(mat.$green-palette, 600);
 
-  $user-theme: mat.define-light-theme($user-primary, $user-accent);
+  $user-theme: mat.define-light-theme(
+    (
+      color: (
+        primary: $user-primary,
+        accent: $user-accent,
+      ),
+    )
+  );
 
   @include mat.all-component-themes($user-theme);
 }
```

Comment:
As I said before, also change **themes** to **colors**, please.
```suggestion
  );

  @include mat.all-component-colors($user-theme);
```
![image](https://user-images.githubusercontent.com/70371638/222359936-c9eb9a49-9d53-409f-9f2d-5b27769e2777.png)